### PR TITLE
Updated WS file for spaces

### LINKED ISSUE #99: Feat: Add WS to spaces
Currently we have added spaces in the new branch `feat/space`
It is implemented with the old approach of HTTP requests
Please add WS support to the spaces feature

### PR DESCRIPTION
**Pull request by Vishawdeep-Singh**
Updated WS file for spaces.
Fixes #99 
Video-:
https://drive.google.com/file/d/1Y-_rSc0lw6LFpbkrAPGuksmr1paSoX_F/view?usp=sharing

### CODE REVIEW

**Comment by Vishawdeep-Singh** [comment]
Resolves: https://github.com/code100x/muzer/issues/99

**Comment by Vishawdeep-Singh** [comment]
@SujithThirumalaisamy

**Comment by SujithThirumalaisamy** [comment]
> @SujithThirumalaisamy 

Thankyou sir. Will review this today and merge this ASAP

File: ws/prisma/migrations/20240828211233_init/migration.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
+-- CreateEnum
```
Comment (SujithThirumalaisamy) [inline ws/prisma/migrations/20240828211233_init/migration.sql:1]: Lets just have the migrations only in next app. Remove it from here. Only run migration in next-app. Only generate client in ws

**Review by SujithThirumalaisamy — COMMENTED** [review]
Also try to cleanup the unused codes. Which are there for the `Oldstreamview` or some redundant code. And also do a lint fix overall

File: next-app/app/dashboard/[spaceId]/page.tsx [hunk 2]
```diff
@@ -14,10 +14,10 @@ import LoadingScreen from "@/components/LoadingScreen";
 export default function Component({params:{spaceId}}:{params:{spaceId:string}}) {
 
 
-  // const { socket, user, loading, setUser, connectionError } = useSocket();
+  const { socket, user, loading, setUser, connectionError } = useSocket();
 
 
-  const [creatorId,setCreatorId]=useState<string>("");
+  const [creatorId,setCreatorId]=useState<string>();
   const [loading1, setLoading1] = useState(true);
```
[the remark this answers is not in the document]
Comment (Vishawdeep-Singh) [reply next-app/app/dashboard/[spaceId]/page.tsx:18]: No i didnt have the type error . But i added a better type

File: ws/prisma/migrations/20240828211233_init/migration.sql (hunk 1 above)
Comment (Vishawdeep-Singh) [reply ws/prisma/migrations/20240828211233_init/migration.sql]: Done

**Comment by Vishawdeep-Singh** [comment]
@SujithThirumalaisamy

**Pull request merged**